build/root/Makefile

```diff
@@ -30,7 +30,7 @@ endif
 #   clean: Clean up.
 
 # It's necessary to set this because some environments don't link sh -> bash.
-SHELL := /bin/bash
+SHELL := /usr/bin/env bash
```

Comment:
I'd be curious to know what lines in the Makefile(s) need this, my understanding is that [make doesn't export this](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html)